source/browseMode.py

```diff
@@ -440,9 +443,56 @@ def _iterNodesByType(self,itemType,direction="next",pos=None):
 	def _iterNotLinkBlock(self, direction="next", pos=None):
 		raise NotImplementedError
 
+	MAX_ITERATIONS_FOR_SIMILAR_PARAGRAPH = 100000

```

Comment:
Just for readability

```suggestion
	MAX_ITERATIONS_FOR_SIMILAR_PARAGRAPH = 100_000
```